application/models/user.go

```diff
@@ -354,3 +350,38 @@ func (u *User) Save() error {
 
 	return DB.Save(u)
 }
+
+func (u *User) getUniqueNickname() error {
```

Comment:
Since the unique nickname isn't a return value, should the name be something like `uniquifyNickname`? Since it doesn't save to the DB, I might be more inclined to keep the name and return the new nickname for the caller to handle. Since it's a non-exported function, either way is probably fine.

Reply:
Done.